### DISCUSSION TITLE
Something wrong about the NEMO_AUSMPLUSM_Rollback branch

### DISCUSSION THREAD

**Discussion by KleinMoretti2018** [block-hczhai/block2-preview#85] [Q&A] [answered]
I compiled this branch in windows10 with  the code python meson.py build and ninja -C build, but there were some errors. 

I tried to compute a example with Euler explicit scheme just using one core, but it was very slow, about 15s to calculate one step, which is obviously not normal.  I used SU2 version 8.0.0 to compute the same example, and it only took about 1s to compute one step.

And I tried to compute this example in parallel, it didn't report any errors, but it got stuck in the terminal,  for a long time the RMS of the first iteration was not printed on the screen.

Is there something wrong with the code in this branch? Or is there something wrong with my compilation？

Thank you in advance for any kind help!

**Comment by hczhai** [comment, marked as answer]
What you described is irrelevant to the ``block2`` code. You should post it in https://github.com/su2code/SU2.
